samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```
@@ -224,10 +227,11 @@ public void onJobModelExpired() {
             } catch (InterruptedException e) {
               Thread.currentThread().interrupt();
               LOGGER.warn(String.format("Shutdown of container: %s for stream processor: %s was interrupted", container, processorId), e);
+            } catch (Exception e) {
```

Why is `Exception` being swallowed?

---

👍